help/line-spacing.md

```diff
@@ -0,0 +1,26 @@
+# Line spacing
+
+Zulip lets you adjust line spacing across the app, to make the UI feel
+comfortable on any screen.
```

Comment:
This isn't about screen shape at all -- but instead about your personal preferences in how your eyes like to read.

Reply:
*As a user*, why would I change this setting, other than the first time? Probably because I'm on a different screen, where it feels different (nobody mentioned screen *shape*), not because my eyes like to read differently now.

Reply:
``` 
Zulip lets you adjust line spacing within paragraphs in addition to                                     
[font size](/help/font-size), so you can configure the reading                                          
experience to be most comfortable for you.                                                              
```